docs/en-US/component/form.md

```diff
@@ -199,6 +199,12 @@ form/accessibility
 | ------- | ------------------------------- | --------------------------- | ------- |
 | trigger | How the validator is triggered. | ^[enum]`'blur' \| 'change'` | —       |
 
+:::tip
+
+If you don't want to trigger the validator based on input events, set the `validate-event` [attribute](/en-US/component/input.html#attributes) as `false` on the `<el-input>` component.
```

Comment:
This attribute is not only owned by el-input, you can give a few more components and use ellipsis.

Reply:
Changed. Thanks.